### PR TITLE
Feanil/edx ecommerce

### PR DESCRIPTION
**Pull request by feanil**
I'll be squashing it before merge but please review.

@e0d @maxrothman @fredsmith 

FYI @rlucioni

### CODE REVIEW

File: playbooks/roles/edx_ecommerce/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  TIME_ZONE: '{{ EDX_ECOMMERCE_TIME_ZONE }}'
+  LANGUAGE_CODE: '{{EDX_ECOMMERCE_LANGUAGE_CODE }}'
+  # email config
+  EMAIL_HOST: '{{ EDX_ECOMMERCE_EMAIL_HOST }}'
+  EMAIL_HOST_PASSWORD: '{{ EDX_ECOMMERCE_EMAIL_HOST_PASSWORD }}'
+  EMAIL_HOST_USER: '{{ EDX_ECOMMERCE_EMAIL_HOST_USER }}'
+  EMAIL_PORT: $EDX_ECOMMERCE_EMAIL_PORT
+  API_AUTH_TOKEN: '{{ EDX_ECOMMERCE_AUTH_TOKEN }}'
+  STATICFILES_DIRS: []
+#  STATICFILES_DIRS: ['{{ edx_ecommerce_static_path }}']
```
Comment (rlucioni) [inline playbooks/roles/edx_ecommerce/defaults/main.yml:63]: @feanil is it necessary to specify `STATICFILES_DIRS` here?

Comment (feanil) [inline playbooks/roles/edx_ecommerce/defaults/main.yml:63]: @rlucioni unfortunately yes.  The way that the config is currently setup, we override everything in production.

File: playbooks/roles/edxlocal/tasks/main.yml [hunk 2]
```diff
@@ -102,6 +102,20 @@
...
+  mysql_db: >
+    db="{{ EDX_ECOMMERCE_DEFAULT_DB_NAME }}"
+    state=present
+    encoding=utf8
+  when: EDX_ECOMMERCE_DEFAULT_DB_NAME is defined
+
+- name: setup users for edx_ecommerce
+  mysql_user: >
+    name="{{ EDX_ECOMMERCE_DEFAULT_DB_NAME }}"
+    password="password"
```
Comment (maxrothman) [inline playbooks/roles/edxlocal/tasks/main.yml]: Could this reference `EDX_ECOMMERCE_DATABASES['PASSWORD']` instead?

**Comment by maxrothman** [comment]
Did you see any performance improvement with the static `ansible_managed` var?

**Comment by feanil** [comment]
@maxrothman I didn't do any performance tests for ansible managed var changes.  I've updated for the other comment, please take a look at the latest commit.

**Comment by maxrothman** [comment]
:+1:

**Pull request merged**